server/models/annotation.py

```diff
@@ -436,8 +491,19 @@ def load(self, id, region=None, getElements=True, *args, **kwargs):
             annotation.
         :returns: the matching annotation or none.
         """
-        annotation = super(Annotation, self).load(id, *args, **kwargs)
-        if annotation is not None and getElements:
+        annotation = super(Annotation, self).load(id, force=True)
```

Comment:
Is there any chance we want to pass through the `args` and `kwargs`, such as the `exc` or `fields` parameters?

Reply:
It's possible... One issue is that the fields parameter could break the assumptions of the migration.  I was trying to avoid having to load the model twice, but maybe it's unavoidable if we want a truly transparent implementation here.

Reply:
Perhaps we would load twice only if the `fields` parameter was specified?

A different approach would be to migrate all of the annotations when the model is instantiated, and then this can just use the superclass load.

Reply:
I do like the idea of making the schema migration a more explicit operation either done on startup or via a command like `django-admin migrate`.  I think this is infrastructure that would be nice to add to girder core though.  Perhaps I'll bring it up as a discussion topic in our girder meeting.

Since this feature is needed sooner, I'll just go with the loading the model twice when necessary.